docs/dev/adding_instruments.rst

```diff
@@ -105,6 +105,13 @@ This is a minimal instrument definition:
 
 Make sure to include the PyMeasure license to each file, and add yourself as an author to the :code:`AUTHORS.txt` file.
 
+There is a certain order of elements in an instrument class that is useful to adhere to:
+
+* First, the initializer (the :code:`__init__()` method), this makes it faster to find when browsing the source code.
+* Then class attributes/variables, if you need them.
+* Then properties (pymeasure-specific or generic Python variants). This will be the bulk of the implementation.
+* Finally, any methods.
```

Comment:
Let us mention somewhere (I think it is fitting here or about properties), that it is perfectly fine to define methods. For example:
```suggestion
* Finally, any methods. 'Actions' like :code:`recall()` or :code:`trigger_scan()` may find their place here.
```

Reply:
I have some drafts for dealing with this already locally.

Reply:
Done.